local-volume/README.md

```diff
@@ -165,30 +165,39 @@ $ kubectl create -f bootstrapper/deployment/kubernetes/example-storageclass.yaml
 
 ### Step 3: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Using the local volume static provisioner 
 
-This is optional, only for automated creation and cleanup of local volumes. See
-[bootstrapper/](./bootstrapper) and [provisioner/](./provisioner) for details and
-sample configuration files.
+	1. Install helm
+See README.md located in ./local-volume/helm directory.
 
-1. Create an admin account with cluster admin priviledge:
+	2. Create an admin account with cluster admin privilege:
 ``` console
-$ kubectl create -f bootstrapper/deployment/kubernetes/<version>/admin-account.yaml
+$ kubectl create -f local-volume/provisioner/deployment/kubernetes/admin_account.yaml  
 ```
+	3. Generate Provisioner's DaemonSet and ConfigMap spec, and customize it.
+In the simplest form, to generate them, you run:
 
-2. Create a ConfigMap with your local storage configuration details.
 ``` console
-$ kubectl create -f bootstrapper/deployment/kubernetes/<version>/example-config.yaml
+helm template local-volume/helm/provisioner > local-volume/provisioner/deployment/kubernetes/provisioner_generated.yaml 
 ```
-**Note**: This configmap is required in order to run the bootstrapper.
 
+This command will generate a ready to use with **kubectl** yaml file. A user can further
```

Comment:
This command will generate yaml files for the provisioner using default values specified in helm/provisioner/values.yaml.  You can modify values.yaml directly, or provide a custom values file:

Reply:
done